src/app/shared/constants.ts

```diff
@@ -38,8 +38,5 @@ export const currentTOSVersion: User.TosversionEnum = User.TosversionEnum.TOSVER
 export const currentPrivacyPolicyVersion: User.PrivacyPolicyVersionEnum = User.PrivacyPolicyVersionEnum.PRIVACYPOLICYVERSION25;
 export const dismissedLatestTOS = 'dismissedLatestTOS';
 export const dismissedLatestPrivacyPolicy = 'dismissedLatestPrivacyPolicy';
-// This is used to toggle whether GitHub apps are recommended or not.
-// If it's to be recommended permanently, it's better to revert this commit instead.
```

Comment:
I'm assuming the commit that added this was looked at and pretty much reverted

Reply:
> I'm assuming the commit that added this was looked at and pretty much reverted

I just double-checked, and yes, pretty much. Although TBH I did by letting the compiler flag errors after I deleted this property -- figured enough may have changed since then.